includes/class-multisite-directory-taxonomy.php

```diff
@@ -133,4 +134,30 @@ public function saveTermGeo ($term_id) {
         }
     }
 
+    /**
+     * Adds a dropdown box to filter directory entries by category in the admin listing
+     *
+     * @link http://www.wpcustoms.net/snippets/admin-filter-for-custom-taxonomies/
+     *
+     * @uses $typenow to detect the post type being listed
+     */
+    public function add_taxonomy_admin_filter () {
+
+        global $typenow;
+
+        if ($typenow == Multisite_Directory_Entry::name) {
+            $tax_obj = get_taxonomy(self::name);
+            $tax_name = $tax_obj->labels->name;
+            $terms = get_terms(self::name);
+            if (count($terms) > 0) {
+                echo '<select name="' . self::name . '" id="' . self::name . '" class="postform">';
+                echo '<option value="">' . __('Show All', 'multisite-directory') . ' ' . $tax_name . '</option>';
+                foreach ($terms as $term) {
+                    echo '<option value="' . $term->slug . '"' . ( $_GET[self::name] == $term->slug ? ' selected="selected"' : '' ) . '>' . $term->name . ' (' . $term->count . ')</option>';
```

Comment:
Fixed! I always forget (or get lazy) about these. It's even worse in cases like this, because the notices get output inside the `<select>` so they're not visible unless you inspect the source code.